services/api-server/src/simcore_service_api_server/api/routes/solvers_jobs.py

```diff
@@ -271,3 +276,54 @@ async def get_job_outputs(
 
     job_outputs = JobOutputs(job_id=job_id, results=results)
     return job_outputs
+
+
+@router.get(
+    "/{solver_key:path}/releases/{version}/jobs/{job_id:uuid}/outputs/logfile",
+    response_class=RedirectResponse,
+    responses={
+        status.HTTP_200_OK: {
+            "content": {
+                "application/octet-stream": {
```

Comment:
I don't understand when this replies with 200 and when with 307. Could you explain that?

Reply:
This is a trick i need to make to produce the client.
The redirection (as indicated in the diagram above) is the s3 pre-signed link which ends up responding 200 with a file.
The client is auto-generated from the OAS and there I need to specify the final "outcome" otherwise the generator cannot resolve what is the returned type. 
I will add some doc to clarify